modules/core/src/main/scala/shop/effects/Background.scala

```diff
@@ -11,11 +11,11 @@ trait Background[F[_]] {
 }
 
 object Background {
```

Comment:
I think Background is now basically `cats.effect.std.Supervisor`

Reply:
although you'll need to `.void` on the fiber :P so we could make a Background implementation using that